packages/amplify-appsync-simulator/src/server/subscription.ts

```diff
@@ -24,7 +25,7 @@ const log = console;
 
 export class SubscriptionServer {
   private registrations;
-  private iteratorTimeout: Map<string, NodeJS.Timeout>;
+  private iteratorTimeout: Map<string, NodeJS.Timer>;
```

Comment:
Is this because of update to typescript/nodejs?

Reply:
Downgrade of Node, yes.